pkg/test/testreconciler/simpletest/testdata/reconcile/direct/create/expected-http.yaml

```diff
@@ -114,6 +114,32 @@ Date: (removed)
 
 ---
 
+GET http://kube-apiserver/api/v1?timeout=32s
```

Comment:
Not entirely sure why this one is changing...  Maybe try updating the golden output with a workspace?

Reply:
Ah - this looks better now :-)